Fix Messages and Tags typings

sradevski opened the pull request:
Updated according to the [API documentation](https://dev.frontapp.com/#send-reply)

Change-type: patch
Signed-off-by: Stevche Radevski <stevche@balena.io>

sradevski committed:
Fix Messages and Tags typings

Change-type: patch
Signed-off-by: Stevche Radevski <stevche@balena.io>

hedss reviewed (commented):
Just to check, do the tests still pass? Certainly in the past, the Front documentation has been wrong and I had to initially test types manually to get the correct responses. :)

I assume we still have a test Front account, but I don't know what the credentials for them are any more (I think Tim changed these).

sradevski commented:
@hedss I checked in previous PRs and it seems they consistently fail. The types I added were already in use in a JavaScript file which I converted to TypeScript (thus the type additions), and the API documentation has them, so they should be valid. It would be nice to get the test account and try it out though.

hedss reviewed (approved):
LGTM as it sounds like this is already working in another source, then. :)
